change/@fluentui-react-charting-20c8ae70-f2a4-4ef4-89b6-7ca932f7481d.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
The issue is still reproing for stacked bar chart.

Reply:
Can you tell me how I can also check this, does it have something to do with the code sandbox?